marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/EntitySearchControllerTest.java

```diff
@@ -94,6 +97,14 @@ void testCRUDOnSavedQuery() throws Exception {
                 ObjectNode response = readJsonObject(result.getResponse().getContentAsString());
                 assertTrue(response.isEmpty());
             });
+
+        // Try save/update/delete without the required role "hub-central-saved-query-writer"
+        setTestUserRoles("hub-central-saved-query-reader");
+        loginAsTestUser();
+
+        postJson(SAVED_QUERIES_PATH, json).andExpect(status().isForbidden());
```

Comment:
There's a helper method in AbstractMvcTest to make this a little easier, and it guarantees that the user's authenticated HttpSession is used - verifyRequestIsForbidden. Please use this instead.